op-node/rollup/derive/batches.go

```diff
@@ -47,7 +47,7 @@ func CheckBatch(ctx context.Context, cfg *rollup.Config, log log.Logger, l1Block
 			log.Error("failed type assertion to SpanBatch")
 			return BatchDrop
 		}
-		if !cfg.IsSpanBatch(batch.Batch.GetTimestamp()) {
+		if !cfg.IsSpanBatch(batch.Batch.GetTimestamp()) || !cfg.IsSpanBatch(batch.L1InclusionBlock.Time) {
```

Comment:
This allows a span batch to start being submitted to L1 prior to the activation time so long as the last frame is in a batch after canyon activates. Noted this on the specs as well, but it seems like we should be requiring the whole batch to be post canyon activation.

Reply:
Answered in [here](https://github.com/ethereum-optimism/optimism/pull/7813#discussion_r1377366055).